entrypoint.sh

```diff
@@ -9,10 +9,14 @@ BUILD_CONTEXT=$5
 # Login to GHCR
 echo ${PERSONAL_ACCESS_TOKEN} | docker login https://ghcr.io -u ${GITHUB_ACTOR} --password-stdin
 
+GITHUB_OWNER=`echo ${GITHUB_REPOSITORY} | cut -d/ -f1`
```

Comment:
Can you add a comment here? Something like: 

```
# GITHUB_REPOSITORY is always org/repo syntax. Get the owner in case it is not the actor (when working in an org)
GITHUB_OWNER=`echo ${GITHUB_REPOSITORY} | cut -d/ -f1`
```